src/pages/auth/SignIn.jsx

```diff
@@ -19,33 +20,29 @@ const SignIn = () => {
             </Link>
           </p>
 
-          <div className="mb-3 flex w-full flex-col gap-1">
-            <label htmlFor="email">Email:</label>
-            <input
-              type="email"
-              name="email"
-              id="email"
-              value={form.email}
-              onChange={handleChange}
-              className="w-full rounded border-2 border-gray p-2"
-            />
-            {error.email && <p className="text-red">{error.email}</p>}
-          </div>
+          <Input
+            id="email"
+            name="email"
+            type="email"
+            onChange={handleChange}
+            value={form.email}
+            error={error.email}
+          >
+            Email
+          </Input>
 
-          <div className="mb-3 flex w-full flex-col gap-1">
-            <label htmlFor="password">Password:</label>
-            <input
-              type="password"
-              name="password"
-              id="password"
-              value={form.password}
-              onChange={handleChange}
-              className="w-full rounded border-2 border-gray p-2"
-            />
-            {error.password && <p className="text-red">{error.password}</p>}
-          </div>
+          <Input
+            id="password"
+            name="password"
+            type="password"
+            onChange={handleChange}
+            value={form.password}
+            error={error.password}
+          >
+            Password
+          </Input>
 
-          <p className="mb-7">
+          <p>
             <Link
               to="/password/reset"
               className="flex w-full flex-col underline"
```

Comment:
Do you really need all these classes for the Link? Shouldn't it be better to only have `underline`?